package.json

```diff
@@ -30,7 +30,9 @@
     "@deepkit/core": "1.0.1-alpha.147",
     "@deepkit/type": "1.0.1-alpha.151",
     "@deepkit/type-compiler": "1.0.1-alpha.150",
+    "@effect/schema": "^0.67.12",
```

Comment:
Please use a pinned version. Renovate will keep it up to date.

Reply:
done